core/src/log/elastic.rs

```diff
@@ -125,28 +130,123 @@ pub struct ElasticDrainConfig {
 /// ```
 pub struct ElasticDrain {
     config: ElasticDrainConfig,
+    error_logger: Logger,
+    logs: Arc<RwLock<Vec<ElasticLog>>>,
```

Comment:
We never call `.read` on this, so it can just be a mutex.

Reply:
Good point!